### ISSUE TITLE
Passing status

### ISSUE THREAD

**Issue by srp33** [srp33/CodeBuddy#94]
We might disagree on this, but if someone submits a solution and it passes, it always shows that it's passing, even if they modify the code to something that doesn't pass. They will receive points, and it will show in the navigation bar that it passes. But it seems non-intuitive to show that it's still passing on the main page. Could we update that message so that it's based on their latest submission? Or maybe a better way is to say something like, "A previous submission passed, but the latest submission does not. You will still receive points for the earlier submission. Use the Submissions tab to view earlier submissions." (Maybe too wordy.) @pjtatlow

**Comment by pjtatlow** [comment]
Haha we don't disagree on this one, it is definitely confusing. What do you think about this?

<img width="865" alt="image" src="https://user-images.githubusercontent.com/4894249/191791425-fbe143dc-0be0-4819-905f-82509c546c39.png">

**Comment by srp33** [comment]
Yes, I like that!
